### ISSUE TITLE
[macOS] Copy Shortcut does not work if TextField outside of WebView has focus

### ISSUE THREAD

**Issue by marcosgcd** [pichillilorenzo/flutter_inappwebview#2380]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When a user has focus on a Flutter TextField and tries to copy text from a WebView, nothing gets copied.

I attempted to fix this issue by adding the following code:

```
onWindowFocus: (
    FocusScope.of(context).unfocus();
},
```

This solution worked for the first copy, but it didn’t consistently resolve the problem afterward.

### Expected Behavior

Copy should work without any extra code outside of InAppWebView

### Steps with code example to reproduce

<details>
<summary>Steps with code example to reproduce</summary>

I could reproduce the issue on with example from: https://github.com/pichillilorenzo/flutter_inappwebview/blob/master/flutter_inappwebview/example/lib/in_app_webiew_example.screen.dart

1. Start The Example project (I used MacOS)
2. Click in Url TextField 
3. Try to copy something from the WebView. Don't click anything else to not remove the focus from the textfield manually 
4. Try paste copied text 

</details>

### Stacktrace/Logs

<details>
<summary>Stacktrace/Logs</summary>

```
<Replace this line by pasting your stacktrace or logs here>
```
</details>

### Flutter version

3.24.3

### Operating System, Device-specific and/or Tool

macOS

### Plugin version

6.1.5

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by pichillilorenzo** [comment]
It seems there is an issue with the native Flutter PlatformView itself on macOS!
The same problem occurs also with the `webview_flutter` plugin, so you should open a new issue to the official Flutter repository.

In the meantime, I was able somehow to create a workaround for it on the native side, but this is something that should be fixed natively on Flutter itself.
This workaround should be available in the next 6.2.0 version

**Comment by pichillilorenzo** [comment]
Released new version `6.2.0-beta.1`.
